src/types/chainParams.ts

```diff
@@ -0,0 +1,11 @@
+export type ChainParams = {
+  chainId: string;
```

Comment:
```suggestion
  chainId: number;
```

Reply:
in the file supportedNetworks.ts there's an object called `CHAIN_PARAMS`. One of the reasons this exists in its current format, with type `ChainParams`, is so if a user selects BNB Chain and doesn't have it on MetaMask, he'll be prompted to add BNB Chain to MetaMask. If `chainId: number`, you get the following error if you select BNB Chain and don't have it on MetaMask:

`"Expected 0x-prefixed, unpadded, non-zero hexadecimal string 'chainId'`

I see 2 options with this:
1. Remove the ability to add a chain like BNB to MetaMask if you don't already have it
2. Keep `chainId: string`

Reply:
![Screenshot 2023-06-03 at 13 58 33](https://github.com/airswap/airswap-web/assets/86911296/90c37158-df98-4fac-8780-dcba262fa6dd)

Reply:
I don't have BNB and get this on 1inch.

Reply:
If you don't have a network on MetaMask then try to switch to it, that screen should pop up. The code `method: "wallet_switchEthereumChain",` in like 56 on ChainSelectionPopover makes that happen